addon/bootstrap.js

```diff
@@ -98,18 +98,20 @@ async function startup(addonData, reason) {
   }
   await studyUtils.startup({reason});
 
-  log.debug(`info ${JSON.stringify(studyUtils.info())}`);
+  console.log(`info ${JSON.stringify(studyUtils.info())}`);
 
-  let value = HomepageStudy.reportValue(studyUtils.studyName);
+  let value = HomepageStudy.reportValue(studyConfig.studyName, studyConfig.isSimulation, studyConfig.rapporPath);
   if (!value) {
     studyUtils.endStudy({reason: "ignored"});
     return;
   }
-  // Send RAPPOR response to Telemetry.
-  studyUtils.telemetry({
-    cohort: value.cohort.toString(),
-    report: value.report
-  });
+  if (!studyConfig.isSimulation){
```

Comment:
Let's assume we're always in the non-simulation mode and just call the simulation code if we're in that mode.

Reply:
Done